src/mantle/framework/database/model/class-model.php

```diff
@@ -28,12 +28,12 @@
  * @mixin \Mantle\Framework\Database\Query\Builder
```

Comment:
Looks like you could remove this `@todo` now https://github.com/alleyinteractive/mantle-framework/pull/139/files#diff-1273adc6b762163f4f03cd56c4750f3d2918d3c031918553300a61ef4b906ddfR26

Reply:
good catch!